docs/web-sdk.md

```diff
@@ -61,6 +61,29 @@ And then loaded in your JavaScript and bundled as you like:
 import '@exmg/livery';
 ```
 
+#### Using with Next.js
```

Comment:
This level of depth of sections will not be included in the TOC navigation to the side. I agree that this also depends on NPM installation, but perhaps you could also consider that implied or explain it when you reference Next.js? I'd say we could nest this either under Installation (or Usage?) as just "Next.js".